Extension/package.nls.json

```diff
@@ -242,7 +242,10 @@
     "c_cpp.configuration.caseSensitiveFileSupport.markdownDescription": { "message": "If set to `default`, the file system of the workspace is assumed to be case insensitive on Windows and case sensitive on macOS or Linux. If set to `enabled`, the file system of the workspace is assumed to be case sensitive on Windows.", "comment": [ "Markdown text between `` should not be translated or localized (they represent literal text) and the capitalization, spacing, and punctuation (including the ``) should not be altered." ] },
     "c_cpp.configuration.enhancedColorization.markdownDescription": { "message": "If enabled, code is colorized based on IntelliSense. This setting only applies if `#C_Cpp.intelliSenseEngine#` is set to `default`.", "comment": [ "Markdown text between `` should not be translated or localized (they represent literal text) and the capitalization, spacing, and punctuation (including the ``) should not be altered." ] },
     "c_cpp.configuration.codeFolding.description": "If enabled, code folding ranges are provided by the language server.",
-    "c_cpp.configuration.commentFormatting.description": "Select whether markdown features will be available in comment formatting. By default, only a subset of markdown features are available.",
+    "c_cpp.configuration.markdownInComments.description": "Select whether markdown features will be available in the hover window. By default, only a subset of markdown formatting will be applied to comments in the hover window.",
```

Comment:
Should we prefer the word "tooltip" over "window" for the hover information?

Reply:
Yes (or another preferred term if you can find one).